package.json

```diff
@@ -65,6 +67,7 @@
     "brfs": "^2.0.2",
     "browserify": "^17.0.0",
     "budo": "^11.6.4",
+    "codecov": "^3.8.1",
```

Comment:
Rather than include this as a dep, we should _just_ install this in the CI. There zero benefit to ever running this locally. 

So can you please do a `yarn global add codecov` in the gh actions file, and just running that after the tests run. Similarity get rid of line 33 here.

Reply:
Love it, thanks! I'll take care of that soon. On that note, we should probably do the same over in the `focus-trap-react` repo since it also has `codecov` listed as a dev dependency: https://github.com/focus-trap/focus-trap-react/blob/master/package.json#L77

I can make the same change over there once we get this MR approved.

Reply:
After a few attempts, I think I finally have this working correctly. The CI pipeline passes anyway. 😅 

These two issues helped me resolve the problem of the global yarn package not being installed properly or not being installed where it could be found:

- https://github.com/yarnpkg/yarn/issues/648
- https://github.com/actions/setup-node/issues/85